lib/semigrp.gi

```diff
@@ -114,6 +114,10 @@ InstallMethod(ViewString, "for a monoid with generators",
 [IsMonoid and HasGeneratorsOfMonoid], _ViewStringForSemigroups);
 
 InstallMethod(ViewString, "for an inverse semigroup with generators",
+[IsInverseSemigroup and HasGeneratorsOfSemigroup],
+_ViewStringForSemigroups);
+
+InstallMethod(ViewString, "for an inverse semigroup with generators",
```

Comment:
This method has the same info string as the previous one - could we make it more unique?